<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRentDialog</class>
 <widget class="QDialog" name="AddRentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>187</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>391</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>391</width>
    <height>187</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dodaj Wypo≈ºyczenie</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>391</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <widget class="QComboBox" name="personComboBox"/>
    </item>
    <item>
     <widget class="QComboBox" name="bookComboBox"/>
    </item>
    <item>
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="addButton">
      <property name="text">
       <string>Dodaj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
